lib/fog/azurerm/models/compute/virtual_machine_sizes.rb

```diff
@@ -0,0 +1,18 @@
+module Fog
+  module Compute
+    class AzureRM
+      class VirtualMachineSizes < Fog::Collection
+        attribute :location
+        model Fog::Compute::AzureRM::VirtualMachineSize
+
+        def all(async = false)
```

Comment:
```suggestion
        def all(async = false)
```
The `async` here is not needed. Please remove it.